client/src/pages/ProfileListing/CoffeCard.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';

```

Comment:
`jsx` file, this client code is written in TypeScript so please create a `.tsx` file and utilize TypeScript for type safety on our props.